tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/FlatBuffersStructsTests.swift

```diff
@@ -144,6 +146,8 @@ func createVec2(x: Float32 = 0, y: Float32 = 0, z: Float32 = 0, color: Color2) -
 }
 
 struct Vec2: Readable {
+    var buffer: ByteBuffer! { __p.bb }
```

Comment:
This could very easily clash if someone adds a field named `buffer`.. I would generally say to not add this at all, and instead just directly use `__p.bb` where needed.

Reply:
I’ll rename it to __buffer__ since we are actually using protocols to imply that there would be an object of type buffer for us to use where needed. And if we add a direct access to __p.buffer that would mean that structs would need to be tables and tables would also be structs.